### DISCUSSION TITLE
Action doesn't reload list when using ActionResponseModal

### DISCUSSION THREAD

**Discussion by salmanhijazi** [laravel/nova-issues#4798] [Ideas & Feature Requests]
- Laravel Version: 9.30.0
- Nova Version: 4.13.0
- PHP Version: 8.1.8

### Description
Hello,

I'm using an inline Action and when it is executed, it reloads the resource list page to show updated data. However, when using a custom action response modal, the list is not reloaded. Is this by design or should the list reload before showing the custom response modal?

I've tried adding emits to my custom asset, `this.$emit('actionExecuted')` and `Nova.$emit('action-executed')`, but that didn't do the trick. 

Thank you in advance!

**Comment by crynobone** [comment]
It is by design at the moment, marking this as a feature request.
